src/clr-angular/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -200,6 +200,14 @@
       );
     }
 
+    &.is-excluded {
+      opacity: 0.25;
```

Comment:
Need to test if this is too low contrast, probably better to set a different background color.